### PR TITLE
[SPACES] Create new "Spaces" view

### PR DESCRIPTION
**Pull request by abelgardep**
## Related Issues
App: https://github.com/owncloud/android/issues/3805

Library PR (if needed): https://github.com/owncloud/android-library/pull/527
_____

## QA

- [ ] (1) Images not updated by pull to refreshing https://github.com/owncloud/android/pull/3843#issuecomment-1378643548

### CODE REVIEW

File: owncloudApp/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -67,4 +67,8 @@
     <!-- Bottom Sheet Fragment -->
     <color name="bottom_sheet_fragment_item_color">#6E758C</color>
     <color name="bottom_sheet_fragment_title_color">#6E758C</color>
-</resources>
\ No newline at end of file
+
+    <!-- Spaces -->
+    <color name="spaces_card_background_color">#edf3fa</color>
+    <color name="spaces_card_default_image_color">#4c5f79</color>
```
Comment (abelgardep) [inline owncloudApp/src/main/res/values/colors.xml:73]: We may need to move them to setup xml because that blue is from oC

Comment (JuancaG05) [reply owncloudApp/src/main/res/values/colors.xml:73]: Ok, moved!

**Review by JuancaG05 — APPROVED** [review]

File: owncloudApp/src/main/java/com/owncloud/android/ui/activity/DrawerActivity.kt [hunk 2]
```diff
@@ -224,6 +224,9 @@ abstract class DrawerActivity : ToolbarActivity() {
         // Allow or disallow touches with other visible windows
         getBottomNavigationView()?.filterTouchesWhenObscured = PreferenceUtils.shouldDisallowTouchesWithOtherVisibleWindows(this)
         if (account != null) {
+            capabilitiesViewModel.capabilities.value?.let {
+                setSpacesVisibilityBottomBar(it.peekContent())
+            }
             capabilitiesViewModel.capabilities.observe(this, Event.EventObserver { uiResult: UIResult<OCCapability> ->
                 setSpacesVisibilityBottomBar(uiResult)
             })
```
Comment (abelgardep) [inline owncloudApp/src/main/java/com/owncloud/android/ui/activity/DrawerActivity.kt:232]: Try this to handle the capabilities value as many times as needed. EventObserver works as OneTime value observer.
```suggestion
            capabilitiesViewModel.capabilities.observe(this) { event: Event<UIResult<OCCapability>> ->
                setSpacesVisibilityBottomBar(event.peekContent())
            }

File: owncloudApp/src/main/res/values/colors.xml [hunk 3]
```diff
@@ -69,6 +69,5 @@
     <color name="bottom_sheet_fragment_title_color">#6E758C</color>
 
     <!-- Spaces -->
-    <color name="spaces_card_background_color">#edf3fa</color>
     <color name="spaces_card_default_image_color">#4c5f79</color>
```
Comment (abelgardep) [inline owncloudApp/src/main/res/values/colors.xml:72]: Should it be brandable too? Maybe we could reuse another color for this divider..

File: owncloudApp/src/main/java/com/owncloud/android/ui/activity/DrawerActivity.kt (hunk 2 above)
Comment (JuancaG05) [reply owncloudApp/src/main/java/com/owncloud/android/ui/activity/DrawerActivity.kt]: True, nice one!

File: owncloudApp/src/main/res/values/colors.xml (hunk 3 above)
Comment (JuancaG05) [reply owncloudApp/src/main/res/values/colors.xml:72]: Ok, I reused another similar one. Anyway, in a beginning this was for the tint of the default icon when the space had no image, but now we print it in black so...

**Comment by jesmrec** [comment]
### (1)

Steps:

1. In web, create space with image
2. Open app > Spaces. Space with the image correctly shown
3. In web, edit image, setting a new one for the space
4. In app, pull to refresh in Spaces view

Current: image not refreshed. Required to browse through tabs to get an update
Expected: Pull to refresh, refreshing image as the title and subtitle do

Samsung A51, Android12
`bb0260d3`

**Pull request merged**